editoast/src/views/paced_train.rs

```diff
@@ -406,4 +496,60 @@ mod tests {
         );
         assert_eq!(paced_train.step, response.paced_train.paced.step.into());
     }
+
+    async fn app_infra_id_paced_train_id_for_simulation_tests() -> (TestApp, i64, i64) {
+        let db_pool = DbConnectionPoolV2::for_tests();
+        let small_infra = create_small_infra(&mut db_pool.get_ok()).await;
+        let timetable = create_timetable(&mut db_pool.get_ok()).await;
+        let rolling_stock =
+            create_fast_rolling_stock(&mut db_pool.get_ok(), "simulation_rolling_stock").await;
+        let paced_train_base: PacedTrainBase = PacedTrainBase {
+            train_schedule_base: TrainScheduleBase {
+                rolling_stock_name: rolling_stock.name.clone(),
+                ..serde_json::from_str(include_str!("../tests/train_schedules/simple.json"))
+                    .expect("Unable to parse")
+            },
+            paced: Paced {
+                duration: Duration::hours(1).try_into().unwrap(),
+                step: Duration::minutes(15).try_into().unwrap(),
+            },
+        };
+        let paced_train: PacedTrainChangeset = paced_train_base.into();
+        let paced_train = paced_train
+            .timetable_id(timetable.id)
+            .create(&mut db_pool.get_ok())
+            .await
+            .expect("Failed to create paced train");
+        let core = mocked_core_pathfinding_sim_and_proj(paced_train.id);
+        let app = TestAppBuilder::new()
+            .db_pool(db_pool.clone())
+            .core_client(core.into())
+            .build();
+        (app, small_infra.id, paced_train.id)
+    }
+
+    #[rstest]
+    async fn paced_train_simulation() {
+        let (app, infra_id, train_schedule_id) =
+            app_infra_id_paced_train_id_for_simulation_tests().await;
+        let request = app.get(
+            format!(
+                "/paced_train/{}/simulation/?infra_id={}",
+                train_schedule_id, infra_id
+            )
+            .as_str(),
+        );
+        app.fetch(request).assert_status(StatusCode::OK);
```

Comment:
This test isn't really testing anything really. If we break the simulation processing at some point, but still return OK, this test won't catch anything even though there is indeed a regression. Better assert on the output of the endpoint, or at least check for some properties on it if it's too lengthy. Otherwise you could use [`insta`](https://insta.rs) for snapshot testing (it's already used in editoast to test the output of derive macros).